trunk/SUAVE/Components/Energy/Networks/Battery_Stopped_Rotor.py

```diff
@@ -0,0 +1,170 @@
+# Battery_Stopped_Rotor.py
+# 
+# Created:  Aug 2017, J. Smart
+
+# !! CURRENTLY NON-FUNCTIONAL !!
```

Comment:
I hope it's functional now?

Reply:
Deleted the file, realized the functionality I planned to add was already available elsewhere.